services/horizon/internal/txsub/results/db/main.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 // DB provides transactio submission results by querying the
```

Comment:
saw a typo in the previous line ;)
```suggestion
// DB provides transaction submission results by querying the
```